Fixed ambiguity in docs, regarding batch generation. (#505)
google/exposure-notifications-server: docs/server_functional_requirements.md

You should schedule a script that generates files for download over the HTTPS
protocol to client devices. The generation of these files are a regular and

> frequent operation (so each device can at least download once a day), we recommend that you

Reviewer: The resulting edit isn't grammatically correct as written.

> frequent operation (so each device can at least download once a day), we recommend that you

Reviewer: ```suggestion
frequent operation (batches should be generated at least once a day), we recommend that you
```
